rotkehlchen/data/all_assets.json

```diff
@@ -8914,6 +8914,15 @@
         "symbol": "PIPL",
         "type": "ethereum token"
     },
+    "PIPT": {
+        "coingecko": "power-index-pool-token",
```

Comment:
The indentation should be the same here. Also since this is not in cryptocompare, you need to add "cryptocompare": "", to make sure cryptocompare is not queried.